documentation/tools/oracle-factory/quick-start.md

```diff
@@ -1,174 +1,88 @@
 # Quick Start
 
-## 1. Setup Environment
+## 1.  Getting started
 
-Ensure you have the necessary environment for running the Oracle Factory SDK.
+### **1.1.  Prerequisites**
 
-### **1.1. Prerequisites**
+Before getting started, ensure that you have the following installed on your system:
 
-Before starting, make sure you have the following installed:
+\- [**Node.js**](https://nodejs.org/en/) version 18 or higher
 
-- [**Node.js**](https://nodejs.org/en/) version 18 or higher
-- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)
+\- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)\
```

Comment:
```suggestion
\- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)
```

Reply:
addressed in https://github.com/iExecBlockchainComputing/documentation-tools/pull/27